contributor/contributors-guide.md

```diff
@@ -570,7 +570,65 @@ For details about governance policies for the Accumulo project view the followin
 - [Lazy Consensus][38]
 - [Voting][39]
 
-
+## Test a Accumulo release
+
+1. Set the release version, ID for staging repo, and alias to configure Maven with temporary settings:
+   ```shell
+   export RC_VERSION=1.9.0
+   export RC_STAGING=1070
+   alias mvn='mvn -s /tmp/accumulo-rc-maven.xml'
+   ```
+1. Create temporary Maven settings
+   ```shell
+   $ cat <<EOF >/tmp/accumulo-rc-maven.xml
+   <settings>
+     <profiles>
+       <profile>
+         <id>accumuloRC</id>
+         <repositories>
+           <repository>
+             <id>accumulorc</id>
+             <name>accumulorc</name>
+             <url>https://repository.apache.org/content/repositories/orgapacheaccumulo-\${env.RC_STAGING}/</url>
+           </repository>
+         </repositories>
+         <pluginRepositories>
+           <pluginRepository>
+             <id>accumulorcp</id>
+             <name>accumulorcp</name>
+             <url>https://repository.apache.org/content/repositories/orgapacheaccumulo-\${env.RC_STAGING}/</url>
+           </pluginRepository>
+         </pluginRepositories>
+       </profile>
+     </profiles>
+     <activeProfiles>
+       <activeProfile>accumuloRC</activeProfile>
+     </activeProfiles>
+   </settings>
+   EOF
+   ```
+1. If a new release candidate is made, update your staging repo ID.
+   ```shell
+   $ export RC_STAGING=1071
```

Comment:
Probably don't need this section. I think it's probably already obvious that the staging number corresponds to the staging repo that you want to test, and should change if you want to test a different staging repo.

Reply:
Removed in c3749ada76db4